libs/shared/ofrep-core/src/lib/provider/ofrep-provider-options.ts

```diff
@@ -21,26 +17,27 @@ export type OFREPProviderBaseOptions = {
    * Optional Headers supplier function.
    * @returns HttpHeaders
    */
-  headersFactory?: () => HttpHeaders;
+  headersFactory?: () => Promise<[string, string][]>;
   /**
    * Optional static headers.
    */
-  headers?: HttpHeaders;
+  headers?: [string, string][];
```

Comment:
This type can be used to init the native fetch headers, so it's a familiar type to people and allows us to leverage existing utilities.

Reply:
Yes definitely, `[string, string][]` was a subtype of it, but it makes sense to have the standard type here.